docs/introduction/developer-quickstart.md

```diff
@@ -143,6 +250,62 @@ How it works:
 
 All you have to do is emit events as usual, and the platform applies common-sense visibility rules.
 
+
+# 3. Building the Frontend
+
+Creating a user-friendly frontend is crucial for interacting with your smart contracts on Obscuro. This section will guide you through setting up a frontend application, installing necessary packages, and integrating the Obscuro network.
+
+## 3.1. Setting Up Your Project
+
+Start by creating a new directory for your frontend application and initialize a Node.js project:
+
+```bash
+mkdir my-obscuro-dapp
+cd my-obscuro-dapp
+npm init -y
+```
+
+## 3.2. Setting Up Your Application
+
+Create your components and set up the application structure. Ensure that you have a component to interact with your smart contract.
+
+## 3.3. Integrating Obscuro Network
+
+### 3.3.1. Using Obscuro Gateway Widget Package
+
+Install the Obscuro Gateway Widget package to easily integrate the Obscuro network into your dApp:
+
+```bash
+npm install @obscuro/obscuro-gateway-widget
+```
+
+Import and use the package in your components to interact with the Obscuro network.
+
+### 3.3.2. Using CDN as a Fallback
+
+In case you are unable to install the Obscuro Gateway Widget package, you can use the Obscuro Gateway Library through CDN. Add the following script tag to your HTML file:
```

Comment:
when would the user be unable to install? 
Let's move this to a troubleshooting page